### PR TITLE
Update Readme.txt vs. 2

### PR DESCRIPTION
**Pull request by MerveGuel**
Readme.txt is updated with a new word

### CODE REVIEW

File: Readme.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 Hi
+Hello
```
Comment (gulmmerve) [inline Readme.txt:2]: Nice work!! 👍 💯

**Review by gulmmerve — APPROVED** [review]
Good job, keep the work up!! 💯

File: Readme.txt (hunk 1 above)
Comment (MerveGuel) [reply Readme.txt:2]: Will do so!

File: Readme.txt [hunk 2]
```diff
@@ -1 +1,2 @@
 Hi
```
Comment (gulmmerve) [inline Readme.txt:1]: Alright, can be merged.

**Review by gulmmerve — APPROVED** [review]
Approved!

File: Readme.txt (hunk 1 above)
Comment (gulmmerve) [inline Readme.txt:2]: Could be better..

Comment (gulmmerve) [inline Readme.txt:2]: Please work on it a bit more.

**Review by gulmmerve — CHANGES_REQUESTED** [review]
Kindly please work a bit more.

**Pull request merged**